packages/xstate-wallet/.env

```diff
@@ -29,8 +29,7 @@ TEST_ASSET_HOLDER2_ADDRESS = 0x0000000000000000000000000000000000000000
 
 # TODO: This should be something the wallet/user decides not a .env variable. 
 # For now this allows integration tests to test ledger funding
-USE_VIRTUAL_FUNDING=TRUE
```

Comment:
Is the removal of environment variables intentional?

Reply:
No, but it does reflect the conditions under which these changes led to a click through test succeeding. Looks like we leave them in place but preserve those conditions in bcabeed.